addons/ekco/template/base/install.sh

```diff
@@ -54,6 +54,11 @@ function ekco_pre_init() {
         LOAD_BALANCER_ADDRESS=localhost
         LOAD_BALANCER_PORT="6444"
     fi
+
+    EKCO_MAINTAIN_MINIO=false
+    if [ -n "$MINIO_VERSION" ] && [ -n "$OPENEBS_LOCALPV" ]; then
+        EKCO_MAINTAIN_MINIO=true
```

Comment:
can you make an ekco property that allows the end-user to disable this functionality as an escape hatch similar to 

```
    EKCO_RECONCILE_CEPH_CSI_RESOURCES=true
    if [ -z "$ROOK_VERSION" ] || [ "$EKCO_ROOK_SHOULD_DISABLE_RECONCILE_CEPH_CSI_RESOURCES" = "1" ]; then
        EKCO_RECONCILE_CEPH_CSI_RESOURCES=false
    fi
```

Reply:
yeah

Reply:
done, `Ekco.MinioShouldDisableManagement`